### PR TITLE
Adds folio login dag and renames symphony login dag

### PR DESCRIPTION
**Pull request by jgreben**
I though that since the login step for both symphony and folio are discreet steps in the request process, they should have their own dags(?) If there is some other logical structure please let me know.

### CODE REVIEW

File: dags/folio_login.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""FOLIO Operators and Functions for Institutional DAGs."""
+import json
+
+# from airflow.operators.bash import BashOperator
```
Comment (jermnelson) [inline dags/folio_login.py:4]: Kruft? Since this and line 5 are commented-out, might as well remove them.

File: dags/folio_login.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# from airflow.operators.python import PythonOperator
+from airflow.providers.http.operators.http import SimpleHttpOperator
+
+
+def FolioLogin(**kwargs) -> SimpleHttpOperator:
+    tenant = kwargs.get("tenant")
+    conn_id = kwargs.get("conn_id")
```
Comment (jermnelson) [inline dags/folio_login.py:11]: I've been documenting in the wiki how to set-up both AWS and Symphony Connections, could you do the same for FOLIO? Thanks!

**Review by jermnelson — CHANGES_REQUESTED** [review]
Looks good. Minor requests to remove commented-out code and add documentation to the wiki on FOLIO Connection.

File: dags/folio_login.py (hunk 1 above)
Comment (jgreben) [reply dags/folio_login.py]: Removed.

File: dags/folio_login.py (hunk 2 above)
Comment (jgreben) [reply dags/folio_login.py:9]: I added a placeholder page with similar text as the Symphony page. To be fleshed out as needed later.

**Review by jermnelson — APPROVED** [review]

**Pull request merged**